src/print.jl

```diff
@@ -140,10 +140,10 @@ function math_symbol(::Type{IJuliaMode}, name::Symbol)
     end
 end
 
-wrap_in_math_mode(str) = "\$\$ $str \$\$"
-wrap_in_inline_math_mode(str) = "\$ $str \$"
+_wrap_in_math_mode(str) = "\$\$ $str \$\$"
+_wrap_in_inline_math_mode(str) = "\$ $str \$"
 
-plural(n) = (n==1 ? "" : "s")
+plural(n) = (n == 1 ? "" : "s")
```

Comment:
Unexport this?

Reply:
I was hesitant since it is used in `JuMPExtension.jl`